wdl/gwas/regenie_recessive_sub.wdl

```diff
@@ -0,0 +1,365 @@
+task step2 {
+
+    String pheno
+    File cov_pheno
+    String covariates
+    String test
+    # regenie accepts either no test specified (i.e. additive), or '--test dominant' or '--test recessive'. '--test additive' is an error.
+    String test_cmd = if test == "additive" then "" else "--test "+ test
+    Boolean is_binary
+    File bgen
+    File bgi = bgen + ".bgi"
+    File sample = bgen + ".sample"
+    String prefix = sub("R8_sisuv4_GRM_V0_LD_0.1_", "_pred.list", "") + pheno + "." + basename(bgen) 
+    Array[File] loco
+    Array[File] nulls
+    Int bsize
+    String options
+
+    String docker
+
+    command <<<
+
+        n_cpu=`grep -c ^processor /proc/cpuinfo`
+
+        # move loco files to /cromwell_root as pred file paths point there
+        for file in ${sep=" " loco}; do
+            mv $file .
+        done
+
+        # move null files to /cromwell_root as firth_list file paths point there
+        for file in ${sep=" " nulls}; do
+            mv $file .
+        done
+
+        
+        echo "${pheno} /cromwell_root/R8_sisuv4_GRM_V0_LD_0.1.${pheno}.loco.gz" > "R8_sisuv4_GRM_V0_LD_0.1.${pheno}.pred.list"
+        echo "${pheno} /cromwell_root/R8_sisuv4_GRM_V0_LD_0.1.${pheno}.firth.gz" >"R8_sisuv4_GRM_V0_LD_0.1.${pheno}.firth.list"
```

Comment:
Seems like this workflow deals with only one pheno at a time, meaning only one loco and firth file at a time. 
Then it could be simplified like so:
```
echo "${pheno} ${loco[0]}" > ${pheno}.pred.list
echo "${pheno} ${nulls[0]}" > ${pheno}.firth.list
```
and if we change loco and nulls to be just File instead of Array[File], the indexing can be taken out:
```
echo "${pheno} ${loco}" > ${pheno}.pred.list
echo "${pheno} ${nulls}" > ${pheno}.firth.list
```
This would also make it work on other releases than R8 and with other GRMs, since now the R8_sisuv4_GRM etc is hardcoded.